examples/lode-linear/environment.yml

```diff
@@ -3,7 +3,6 @@ channels:
 dependencies:
   - python=3.11
   - pip
-  - rust >=1.65
   - pip:
     - ase==3.22.1
     - equisolve @ git+https://github.com/lab-cosmo/equisolve.git@c858bedef4b2799eb445e4c92535ee387224089a
```

Comment:
we should get rid of this dependency, nobody will be able to use it …

Reply:
I very much agree, but I think this, as well as the older Hamiltonian examples, should just be "retired" as soon as we have good examples based on the more modern technology. for the moment I'd leave them be, certainly for the purpose of this PR that is just a superficial cleaning of stuff that relies on commit pinning for reasons that are no longer relevant